src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,82 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
```

Comment:
Ensure that the `getUser` function is correctly defined in the `../../api` module. It should return a promise that resolves to a user object based on the `userId`.